docs/cluster-agent/CUSTOM_METRICS_PROVIDER.md

```diff
@@ -10,32 +10,32 @@ As of Kubernetes v1.6, it is possible to autoscale off of [custom metrics](https
 Custom metrics are user defined and are collected from within the cluster.
 As of Kubernetes v1.10, support for external metrics was introduced so users can autoscale off of any metric from outside the cluster that is collected for you by Datadog.
 
-The custom and external metric provider, as opposed to the metrics server, are resources that have to be implemented and registered by the user.
+The custom and external metric providers, as opposed to the metrics server, are resources that have to be implemented and registered by the user.
 
-As of v1.0.0, the DCA implements the External Metrics Provider interface.
+As of v1.0.0, the Datadog Cluster Agent implements the Custom Metrics Provider interface for External Metrics.
```

Comment:
`As of v1.0.0, the Custom Metrics Server in the Datadog Cluster Agent implements the External Metrics Provider interface for External Metrics`